docs/pages/enterprise/getting-started.mdx

```diff
@@ -214,12 +214,33 @@ role only allows SSH logins as `root@host`.
 ```yaml
```

Comment:
In addition to this, I think it would also help to include the table of preset users from the Access Controls reference here so users don't need to navigate to the "presets" link to see the table. Since the table is short, viewing it here should take less time/cognitive effort than navigating to a different page.